modules/openapi-generator-maven-plugin/examples/java-client.xml

```diff
@@ -39,6 +40,36 @@
                             <library>jersey2</library>
                         </configuration>
                     </execution>
+                    <execution>
```

Comment:
I added the testcase you mentioned https://github.com/OpenAPITools/openapi-generator/pull/3826#issuecomment-528736638.
But I couldn't reproduce the error in my environment.
```
mvn clean compile -f modules/openapi-generator-maven-plugin/examples/java-client.xml

[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.openapitools:sample-project >-------------------
[INFO] Building sample-project 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ sample-project ---
[INFO] Deleting /mnt/c/publicgithub/openapi-generator/modules/openapi-generator-maven-plugin/examples/target
[INFO]
[INFO] --- openapi-generator-maven-plugin:4.1.2-SNAPSHOT:generate (default) @ sample-project ---
[WARNING] Output directory does not exist, or is inaccessible. No file (.openapi-generator-ignore) will be evaluated.
[INFO] OpenAPI Generator: java (client)
[INFO] Generator 'java' is considered stable.
......
......
[INFO] --- openapi-generator-maven-plugin:4.1.2-SNAPSHOT:generate (remote) @ sample-project ---
[WARNING] Output directory does not exist, or is inaccessible. No file (.openapi-generator-ignore) will be evaluated.
[INFO] OpenAPI Generator: java (client)
[INFO] Generator 'java' is considered stable.
......
......
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ sample-project ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /mnt/c/publicgithub/openapi-generator/modules/openapi-generator-maven-plugin/examples/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.6.1:compile (default-compile) @ sample-project ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 46 source files to /mnt/c/publicgithub/openapi-generator/modules/openapi-generator-maven-plugin/examples/target/classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 49.616 s
[INFO] Finished at: 2019-09-06T16:24:45+09:00
[INFO] ------------------------------------------------------------------------
```

Reply:
Can you check your error will be reproduced with my latest commit including testcase?
```
mvn clean install
mvn clean compile -f modules/openapi-generator-maven-plugin/examples/java-client.xml
```

Reply:
Also waiting for who Shippable CI will judge the testcase.

Reply:
I pull the latest again and no longer seeing the issue. Must be the setup in my end.

Thanks for adding the test case.

I'll merge after all CI tests pass.